java-rest-service/config/testcontainers-test-pipeline.yaml

```diff
@@ -3,7 +3,7 @@ kind: Pipeline
 metadata:
   name: java-test-pipeline
```

Comment:
I thought we were removing all test pipeline configuration from the accelerator.

Reply:
it is only in the repo, not part of the generated code